### DISCUSSION TITLE
How to ignore a specific package's dependency dashboard warning for `Failed to look up` ? The `ignoreDeps` for the package still persists the warning

### DISCUSSION THREAD

**Discussion by nitrocode** [renovatebot/renovate#30889] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I want to ignore a specific package's warning because I've been able to look it up using a `customManager`.

When I use `ignoreDeps`, it still shows the warning in the dashboard.

I want to still see other warnings, just not the warning for this particular package.

### Logs (if relevant)

N/A

**Comment by rarkins** [comment, marked as answer]
Use a package rule to match against manager and packageName and then set enabled=false to prevent the lookup in the first place

**Reply by nitrocode** [reply]
Thanks, that did the trick!
